### PR TITLE
build: enable `verbatimModuleSyntax`

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

File: e2e/utils.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import net from 'node:net';
-import { ConsoleMessage, test as basicTest } from '@playwright/test';
+import { type ConsoleMessage, test as basicTest } from '@playwright/test';
```
Comment (dai-shi) [inline e2e/utils.ts:2]: I personally prefer separate lines:
```suggestion
import { test as basicTest } from '@playwright/test';
import type { ConsoleMessage } from '@playwright/test';
```

**Comment by dai-shi** [comment]
Can you fix `packages/waku/src/node-loader.ts` too?

**Comment by himself65** [comment]
why ci no error

**Comment by dai-shi** [comment]
It's a warning on line 8&9: `'node:module' imported multiple times`

File: packages/waku/src/node-loader.ts [hunk 2]
```diff
@@ -6,9 +6,9 @@
  *      Refs: https://github.com/nodejs/node/issues/50885
  */
 import type { LoadHook } from 'node:module';
```
Comment (dai-shi) [inline packages/waku/src/node-loader.ts:8]:
```suggestion
import type { LoadHook, ResolveHook } from 'node:module';
```

File: packages/waku/src/node-loader.ts [hunk 3]
```diff
@@ -6,9 +6,9 @@
...
-import { ResolveHook } from 'node:module';
+import type { ResolveHook } from 'node:module';
```
Comment (dai-shi) [inline packages/waku/src/node-loader.ts:9]:
```suggestion
```

File: packages/waku/src/node-loader.ts [hunk 4]
```diff
@@ -5,10 +5,9 @@
  *      which will lead to a React warning on Node.js 20.
  *      Refs: https://github.com/nodejs/node/issues/50885
  */
-import type { LoadHook } from 'node:module';
-import { ResolveHook } from 'node:module';
+import type { LoadHook, ResolveHook } from 'node:module';
```
Comment (dai-shi) [inline packages/waku/src/node-loader.ts:8]:
```suggestion
import type { LoadHook, ResolveHook, LoadFnOutput, LoadHookContext } from 'node:module';
```

File: packages/waku/src/node-loader.ts [hunk 5]
```diff
@@ -5,10 +5,9 @@
...
 import * as RSDWNodeLoader from 'react-server-dom-webpack/node-loader';
-import { LoadFnOutput, LoadHookContext } from 'module';
+import type { LoadFnOutput, LoadHookContext } from 'module';
```
Comment (dai-shi) [inline packages/waku/src/node-loader.ts:10]:
```suggestion
```

**Review by dai-shi — APPROVED** [review]

**Pull request merged**